style.css

```diff
@@ -0,0 +1,66 @@
+*{
+   /* background-image: url(./survey2.jpg); */
```

Comment:
Please do not leave unused code in your codebase. Unused code makes code unclean